### PR TITLE
 #164 issue : Add PMD as community partner

### PR DESCRIPTION
**Pull request by Nareshchoudhary02**
#164 Added PMD as community partner

### CODE REVIEW

**Review by Wabri — CHANGES_REQUESTED** [review]
Rebase your branch to our main branch

**Comment by Nareshchoudhary02** [comment]
> Rebase your branch to our main branch

how to do now I don't understand that !

File: components/SponsorTable.tsx [hunk 1]
```diff
@@ -82,6 +82,9 @@ const SponsorTable = ({isVisible} : TSponsorTable) => {
                     <a href="https://pointerpodcast.it/" target="_blank" rel="noreferrer">
                     <Image width={100} height={100} src="/pointerpodcast_logo.png" alt="Pointer Podcast" />
                     </a>
+                    <a href="https://www.productmanagementday.com/" target="_blank" rel="noreferrer">
+                    <Image width={100} height={100} src="/productmanagementday_logo.png" alt="PMD" />
```
Comment (Wabri) [inline components/SponsorTable.tsx:86]: Let's try to zoom in the image to see if it comes better
```suggestion
                    <Image width={150} height={150} src="/productmanagementday_logo.png" alt="PMD" />
```

**Review by Wabri — CHANGES_REQUESTED** [review]

**Comment by Wabri** [comment]
To rebase your repository you can use the github gui directly to your fork repo page:
Screenshot 2023-01-26 at 17 59 15

Or locally here the docs https://git-scm.com/book/en/v2/Git-Branching-Rebasing 😃  if you need assistance just ask!

**Comment by Wabri** [comment]
Ok rebase not needed, you already done it!

**Comment by Nareshchoudhary02** [comment]
> Ok rebase not needed, you already done it!
Screenshot 2023-01-26 at 10 34 37 PM
I have synchronized branch is it now ok or I am doing something wrong ?

**Review by Wabri — APPROVED** [review]

**Comment by Wabri** [comment]
GTM

**Pull request merged**